app/main.py

```diff
@@ -2,0 +1,31 @@
+import os
+
+
+def move_file(command: str) -> None:
+    components = command.split()
+    first_part, second_part, third_part = components
+    if len(components) == 3 and first_part == "mv":
+
+        file_name = second_part
+        path = third_part
+        dirs = third_part.split("/")
+
+        if len(dirs) > 1:
+            current_path = dirs[0]
+            os.makedirs(current_path, exist_ok=True)
+
+            for folder in dirs[1:-1]:
+                if folder is not None:
```

Comment:
The condition `if folder is not None:` is unnecessary because `dirs[1:-1]` will not contain `None` values. You can safely remove this condition.